content/zh/blog/kubeedge-deployment-manual/index.md

```diff
@@ -0,0 +1,569 @@
+        beego.Router("/track/control/:trackId", new(controllers.TrackController), "get,post:ControlTrack")
+
+        beego.Run(":8089")
+}
+```
+
+**2）构建镜像**
+
+注意：构建镜像时，请将源码拷贝到GOPATH对应的路径下，如果开启了go mod请关闭。
+```
+[root@ke-cloud web-controller-app~]# make all
+[root@ke-cloud web-controller-app~]# make docker
+```
+
+**3）部署web-controller-app**
+```
+[root@ke-cloud ~]# cd $GOPATH/src/github.com/kubeedge/examples/kubeedge-counter-demo/crds
+[root@ke-cloud crds~]# kubectl apply -f kubeedge-web-controller-app.yaml
+```
+
+## 4.3 部署边缘端应用
+边缘端的pi-counter-app应用受云端应用控制，主要与mqtt服务器通信，进行简单的计数功能。
+
+**1）修改代码与构建镜像**
+
+需要将Makefile中的GOARCH修改为amd64才能运行该容器。
+```
+[root@ke-cloud ~]# cd $GOPATH/src/github.com/kubeedge/examples/kubeedge-counter-demo/counter-mapper
+[root@ke-cloud counter-mapper~]# vim Makefile
+.PHONY: all pi-execute-app docker clean
+all: pi-execute-app
+
+pi-execute-app:
+        GOARCH=amd64 go build -o pi-counter-app main.go
+
+docker:
+        docker build . -t kubeedge/kubeedge-pi-counter:v1.0.0
+
+clean:
+        rm -f pi-counter-app
+
+[root@ke-cloud counter-mapper~]# make all
+[root@ke-cloud counter-mapper~]# make docker
+```
+
+**2）部署Pi Counter App**
+```
+[root@ke-cloud ~]# cd $GOPATH/src/github.com/kubeedge/examples/kubeedge-counter-demo/crds
+[root@ke-cloud crds~]# kubectl apply -f kubeedge-pi-counter-app.yaml
+
+说明：为了防止Pod的部署卡在`ContainerCreating`，这里直接通过docker save、scp和docker load命令将镜像发布到边缘端
+[root@ke-cloud ~]# docker save -o kubeedge-pi-counter.tar kubeedge/kubeedge-pi-counter:v1.0.0
+[root@ke-cloud ~]# scp kubeedge-pi-counter.tar root@192.168.1.56:/root
+[root@ke-edge1 ~]# docker load -i kubeedge-pi-counter.tar
+```
+
+## 4.4 体验Demo
+现在，KubeEdge Demo的云端部分和边缘端的部分都已经部署完毕，如下：
+```
+[root@ke-cloud ~]# kubectl get pods -o wide
+NAME                                    READY   STATUS    RESTARTS   AGE     IP             NODE       NOMINATED NODE   READINESS GATES
+kubeedge-counter-app-758b9b4ffd-f8qjj   1/1     Running   0          26m     192.168.1.66   ke-cloud   <none>           <none>
+kubeedge-pi-counter-c69698d6-rb4xz      1/1     Running   0          2m      192.168.1.56   ke-edge1   <none>           <none>
+```
+
+我们现在开始测试一下该Demo运行效果：
+
+**1）执行ON命令**
+
+在web页面上选择ON，并点击Execute，可以在edge节点上通过以下命令查看执行结果：
+```
+[root@ke-edge1 ~]# docker logs -f counter-container-id
+```
+![image](https://github.com/Poorunga/website/blob/wangjiezhang/content/zh/blog/kubeedge-deployment-manual/images/trun_on.png)
+
+**2）查看counter STATUS**
+
+在web页面上选择STATUS，并点击Execute，会在Web页面上返回counter当前的status，如下所示：
+![image](https://github.com/Poorunga/website/blob/wangjiezhang/content/zh/blog/kubeedge-deployment-manual/images/check_status.png)
+
+**2）执行OFF命令**
+
+在web页面上选择OFF，并点击Execute，可以再edge节点上通过以下命令查看执行结果：
+```
+[root@ke-edge1 ~]# docker logs -f counter-container-id
+```
+![image](https://github.com/Poorunga/website/blob/wangjiezhang/content/zh/blog/kubeedge-deployment-manual/images/trun_off.png)

```

Comment:
Keep the images in the same folder or place them in the `static/img` folder and reference them using the `figure` shortcode like this:
`{{< figure src="<file_name>" title="your_title_here" >}}`
You can use the markdown way to reference images as well (as you have done) but no need to use absolute URLs unless the images are being served off a different server.